src/types/note.ts

```diff
@@ -50,6 +50,7 @@ type EditableNoteAttributes = {
   columnId: string;
   parentId: string;
   text: string;
+  hidden: boolean;
```

Comment:
I guess that's a leftover from hide notes

Reply:
Right. Thank you!